autopush/web/base.py

```diff
@@ -19,14 +20,90 @@
     401: "Unauthorized",
     404: "Not Found",
     413: "Payload Too Large",
+    418: "I'm a teapot",
```

Comment:
We generate 418's via our log check. We now check status messages via a handler that looks in this table, so...